sbc_compassion/wizards/import_letters_test.py

```diff
@@ -61,7 +61,7 @@ def _analyze_attachment(self, file_, filename):
             })
             letters_line.letter_image = self.env[
                 'ir.attachment'].create(document_vals[i])
-            letters_line.check_status()
+            letters_line._compute_check_status()
```

Comment:
I'm not sure if its the right way that access to a private methode

Reply:
It's not a problem from Odoo code to access private methods of Odoo objects. Private methods won't be accessible remotely with XML-RPC.